modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1870,6 +1870,60 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12700 Verify that is not possible to view the accordion 'Show Option' in case the product has not options"
```

Comment:
```suggestion
	@description = "COMMERCE-12700 Verify that is not possible to view the accordion 'Show Option' in case the product has no options"
```